changelogs/unreleased/5519-wenterjoy

```diff
@@ -0,0 +1,34 @@
+add new RestoreItemAction of "velero.io/change-image-repository" to handle the issue mentioned at #5519
+
+
```

Comment:
Documentation doesn't really belong in the changelog entries. Just keep the above line and everything below should be moved to docs. You'll want to add a section to site/content/docs/main/restore-reference.md similar to the existing "Changing PV/PVC Storage Classes" section.

Reply:
thanks for your comments and advice

Reply:
@sseago  site/content/docs/main/restore-reference.md updated.